routes/web.php

```diff
@@ -42,12 +42,10 @@
 
 Route::get('styleguide', 'StyleGuideController@index')->name('styleguide');
 
-Route::post('map/hide', function (Request $request) {
-    $request->session()->put('show_map', false);
+Route::post('settings', function(Request $request) {
+    $time = 60 * 1; // in minutes
+    if ($request->has('hide_news_ticker')) Cookie::queue('hide_news_ticker', $request->input('hide_news_ticker'), $time);
+    if ($request->has('show_map')) Cookie::queue('show_map', $request->input('show_map'), $time);
     return response(null, Response::HTTP_OK);
```

Comment:
BTW, this is same as
```suggestion
    return response();
```